src/container/srv_container.c

```diff
@@ -737,6 +738,14 @@ cont_create_prop_prepare(struct ds_pool_hdl *pool_hdl,
 		entry_def->dpe_val = pool_hdl->sph_pool->sp_rp_pda;
 	}
 
+	entry_def = daos_prop_entry_get(prop_def, DAOS_PROP_CO_PERF_DOMAIN);
+	if (pool_hdl->sph_global_ver > 0)
```

Comment:
I think PDA is not in release 2.4, so we might need bump DAOS_POOL_GLOBAL_VERSION to 3, and then here it should.

if (pool_hdl->sph_global_ver > 2)
        D_ASSERT(...)

Reply:
OK, refreshed as the suggestion. Thanks!